### DISCUSSION TITLE
Custom Adjustments

### DISCUSSION THREAD

**Discussion by InstantZzz** [fh-fvtt/zweihander#187] [Ideas]
I'm going through character creation and my character has the _Prostitute_ profession which has the trait _The Full Monty_. The Full Monty reads "All Skill Ranks you acquire in the Charm Skill modify your Base Chance by +20, instead of +10." I imagine this would be quite difficult to implement so perhaps just adding a way for people add a custom bonus to their skills instead of having to remember every time they use said skill to change the Base Chance Modifier.

Something else I haven't been able to spot is how to use the character Upbringing for RP discounts on Skill Focuses. So means of manually spending points for that and additional languages.

**Comment by Re4XN** [comment]
> The Full Monty

Support for this kind of Talent / Trait will come with the implementation of Active Effects. Until then, we have to ask you to hold tight, unfortunately.

> how to use the character Upbringing for RP discounts on Skill Focuses

Skill Focuses are implemented through Unique Advances. The Unique Advance item has a field you can edit where you can input a custom RP cost. For Focuses you have a discount in, just input 50 instead of 100, for example.

**Reply by InstantZzz** [reply]
Alright, thank you for the info.
